.asf.yaml

```diff
@@ -35,6 +35,13 @@ github:
     dev:
       required_status_checks:
         strict: true
+        context:
+          - check-license
+          - CheckDeadLinks
+          - Build
```

Comment:
I find out we have lot of ci job named `Build` should we first rename them?

Reply:
Oh, I got it, this build is for auto license, and the others is in`Build (8)` and `Build (11)`

Reply:
I get this, and close it